core-bundle/tests/Picker/TablePickerProviderTest.php

```diff
@@ -497,7 +501,7 @@ private function createMenuTableProvider(array $modules, string $current, ItemIn
             ->expects($this->exactly(\count($expectedItems)))
             ->method('createItem')
             ->withConsecutive(...$expectedItems)
-            ->willReturn($menu ?? $this->createMock(ItemInterface::class))
+            ->willReturn($menu)
```

Comment:
Why? `$menu` can be `null`, so we cannot change this.

Reply:
It is set a few lines up in the `else` block.